crates/gateway-types/src/request.rs

```diff
@@ -261,9 +261,12 @@ pub mod add_transaction {
         pub max_fee: Fee,
         #[serde_as(as = "Vec<TransactionSignatureElemAsDecimalStr>")]
         pub signature: Vec<TransactionSignatureElem>,
-        pub nonce: TransactionNonce,
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        pub nonce: Option<TransactionNonce>,
 
         pub sender_address: ContractAddress,
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        pub entry_point_selector: Option<EntryPoint>,
```

Comment:
Maybe add a note that these are optional due to invoke v0.

Reply:
Done.